torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -292,6 +323,17 @@ def resize_bounding_box(
     )
 
 
+@register_kernel(resize, datapoints.BoundingBox, datapoint_wrapping=False)
+def _resize_bounding_box_dispatch(
+    bounding_box: datapoints.BoundingBox, size: List[int], max_size: Optional[int] = None, **kwargs
+):
```

Comment:
Here is a case where both requirements 1. and 2. are not met. Meaning, we have implement a intermediate layer and we have to handle wrapping ourselves. To indicate the latter should not happen automatically, we set `datapoint_wrapping=False` during registering.

Reply:
Hmmm it's a bit confusing to see `datapoint_wrapping=False` and then see the wrapping / unwrapping happening just below.